src/main/java/fruitshop/strategy/handlers/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package fruitshop.strategy.handlers;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+import fruitshop.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void calculateOperation(FruitTransaction transaction) {
+        Storage.getStorage().put(transaction.getFruit(), transaction.getQuantity());
```

Comment:
Again I would recommend interacting with storage through dao layer (same for other classes) (but it's up to you)